developers/weaviate/config-refs/env-vars/index.md

```diff
@@ -1,18 +1,24 @@
 ---
 title: Environment variables
-sidebar_position: 40
+sidebar_position: 0
 image: og/docs/configuration.jpg
 # tags: ['HNSW']
 ---
 
-To configure Weaviate in a [Docker](../installation/docker-compose.md) or a [Kubernetes](../installation/kubernetes.md) deployment, set these environment variables
+To configure Weaviate in a [Docker](../installation/docker-compose.md) or a [Kubernetes](../installation/kubernetes.md) deployment, you can set the following environment variables.
```

Comment:
For references, the previous style is preferable imo. 

It is more concise, and the imperative voice is clearer.